solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -369,6 +371,48 @@ contract StandardPoolConverter is
         return reserveBalances(1, 2);
     }
 
+    /**
+     * @dev returns the balance fee of a given reserve token
+     *
+     * @param _reserveToken    reserve token contract address
+     *
+     * @return the balance fee of the given reserve token
+     */
+    function reserveBalanceFee(IERC20 _reserveToken) public view returns (uint256) {
```

Comment:
I don't see why this should be public

Reply:
@yudilevi : I added this function as part of (i.e., in favor of) adding tests for the fee before/after calling `transferFee`.
It is not even used in the contract (i.e., it is currently used **only** for testing).
We can move it to the corresponding helper contract, but the question is - why not keep it?
Just like the set of functions that we have for the reserve balances, don't we want one for the reserve balance fees?

Reply:
If this is only used in tests, you should move it to a `TestStandardPoolConverter.sol` helper instead

Reply:
This is true for pretty much every external function, as well as for several public functions.
The question is as I stated it above.